assets/stories/skate-components/detours/diversionPage.stories.tsx

```diff
@@ -14,6 +15,7 @@ const meta = {
   args: {
     // Provide default route settings
     ...DiversionPanelMeta.args,
+    shape: route39shape,
```

Comment:
Nit: It's throwing me off a bit that the route shape is the 39, but the pill in the story says "66".
```suggestion
    routeName: "39",
    shape: route39shape,
```